rhconsulting_customization_templates.rake

```
@@ -4,15 +4,48 @@ class CustomizationTemplateImportExport
   class ParsedNonDialogYamlError < StandardError; end
 
   def import(filename)
-    raise "Must supply filename" if filename.blank?
-    customization_templates = YAML.load_file(filename)
-    import_customization_templates(customization_templates)
+    raise "Must supply filename or directory" if filename.blank?
+    if File.file?(filename)
+      customization_templates = YAML.load_file(filename)
+      import_customization_templates(customization_templates)
+    elsif File.directory?(filename)
+      Dir.glob("#{filename}/*.yaml") do |fname|
+        customization_templates = YAML.load_file(fname)
+        import_customization_templates(customization_templates)
+      end
+    else
+      raise "Argument is not a filename or directory"
+    end
   end
 
   def export(filename)
-    raise "Must supply filename" if filename.blank?
-    customization_templates_hash = export_customization_templates(CustomizationTemplate.where("system is not true").order(:id).all)
-    File.write(filename, customization_templates_hash.to_yaml)
+    raise "Must supply filename or directory" if filename.blank?
+    begin
+      file_type = File.ftype(filename)
+    rescue
+      # If we get an error back assume it is a filename that does not exist
+      file_type = 'file'
```

Shouldn't we be logging a message here instead of swallowing the exception?